src/molecules/OverlayingPlugLabels.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+import { getVariable } from '../utils';
+
+import { OverlayingPlugLabel } from '../atoms/OverlayingPlugLabel';
+
+const Footer = styled.footer`
```

Comment:
footer? its for seo or smth?

Reply:
I needed an enclosing element to avoid individual labels taking part in the PlugLink's column-direction flex. (The PlugLink's column-direction flex is there to give us the the `order` attribute.) When adding the extra element anyways, we might as well try to add some semantics.

> The footer element represents a footer for its nearest ancestor sectioning content or sectioning root element.

:point_up: HTML spec. It's nearest ancestor sectioning content would be our `<article>` element.

> A footer typically contains information about its section such as who wrote it, links to related documents, copyright data, and the like.

Basically, a `<footer>` is a good place to put metadata that does not go in a `<header>`.